src/portal-components/nf/PopularSearches.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { parseEntityIdFromSqlStatement } from 'synapse-react-client/dist/utils/functions/sqlFunctions'
+import { SynapseClient, SynapseConstants } from 'synapse-react-client'
+import { QueryBundleRequest, RowSet } from 'synapse-react-client/dist/utils/synapseTypes'
+import { gotoExploreToolsWithFullTextSearch } from './BrowseToolsPage'
+
+export type PopularSearchesProps = {
+  sql: string
+}
+
+/**
+ * Expects table/view column "displayText", and "fullTextSearch".  Create links using this information.
+ */
+const PopularSearches: React.FunctionComponent<PopularSearchesProps> = ({
+  sql,
+}) => {
+  const { accessToken } = useSynapseContext()
+  const [rowSet, setRowSet] = useState<RowSet>()
+  const [isLoading, setIsLoading] = useState<boolean>()
+  let mounted = true
+  useEffect(() => {
```

Comment:
This looks fine, but does SRC export `useQueryResultBundle`? Would simplify a lot here.